app/src/main/java/com/infomaniak/mail/ui/main/newMessage/NewMessageActivity.kt

```diff
@@ -115,7 +116,7 @@ class NewMessageActivity : ThemedActivity() {
             trackNewMessageEvent("sendWithoutSubject")
             createDescriptionDialog(
                 title = getString(R.string.emailWithoutSubjectTitle),
-                description = getString(R.string.emailWithoutSubjectDescription),
+                description = getString(R.string.emailWithoutSubjectDescription).toSpanned(),
```

Comment:
I would have created another helper function with takes a string instead of a spanned